plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/PredicatePushdownController.java

```diff
@@ -30,7 +30,7 @@
         }
         return new DomainPushdownResult(domain, Domain.all(domain.getType()));
     };
-    PredicatePushdownController PUSHDOWN_AND_KEEP = (session, domain) -> new DomainPushdownResult(
+    PredicatePushdownController CASE_INSENSITIVE_PUSHDOWN = (session, domain) -> new DomainPushdownResult(
```

Comment:
> Rename PredicatePushdownController to match semantic

This commit does not rename PredicatePushdownController

Reply:
the `PredicatePushdownController` interface is type-agnostic and "case-insensitive" is applicable to char/varchar type only, so maybe:

CASE_INSENSITIVE_PUSHDOWN -> eg CASE_INSENSITIVE_CHARACTER_PUSHDOWN